test/fixtures/github_actions_test.rb

```diff
@@ -0,0 +1,18 @@
+require 'bundler/setup'
+require 'minitest/autorun'
+require 'minitest/reporters'
+require 'minitest/reporters/mean_time_reporter'
```

Comment:
Why is this mean time reporter required here?

Reply:
No idea, it's been a hot minute. I have lost interest for this and don't feel like investigating unless there is some feedback from maintainers, at which point I'll be happy to get this in a shape they feel comfortable merging.